cricPredict/Groups/views.py

```diff
@@ -1,31 +1,62 @@
-from django.http import HttpResponse
-from django.shortcuts import render, redirect
-from .forms import GroupCreationForm
-from django.views.generic import CreateView, ListView
+from django.shortcuts import get_object_or_404
+from rest_framework import status
+from django.views.generic import ListView
+from rest_framework.exceptions import ParseError
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
 from .models import Group
+from User_profile.models import User
+from .serializers import GroupSerializer
 
 
 # Create your views here.
-class CreateGroup(CreateView):
-    model = Group
-    form_class = GroupCreationForm
-    template_name = 'create_group.html'
+class GroupAPIView(APIView):
+    permissions = (IsAuthenticated,)
```

Comment:
once we move to DRF, we should use `authentication_classes` and `permission_classes` instead